keyboards/latin47ble/info.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
This file goes in the VIA repo, not here.

Reply:
Thanks